src/content/app/genome-browser/hooks/useBrowserRouting.test.tsx

```diff
@@ -63,32 +59,43 @@ jest.mock(
   })
 );
 
+const humanGenomeInfo = {
+  assembly_name: 'GRCh38.p13',
+  common_name: 'Human',
+  genome_id: 'homo_sapiens_GCA_000001405_28',
+  scientific_name: 'Homo sapiens',
+  url_slug: 'grch38'
+};
+
+const wheatGenomeInfo = {
+  assembly_name: 'IWGSC',
+  common_name: null,
+  genome_id: 'triticum_aestivum_GCA_900519105_1',
+  scientific_name: 'Triticum aestivum',
+  url_slug: 'iwgsc'
+};
+
 const committedHuman = {
   ...createSelectedSpecies(),
-  genome_id: 'human'
+  genome_id: humanGenomeInfo.genome_id,
+  url_slug: humanGenomeInfo.url_slug
 };
 
 const committedWheat = {
   ...createSelectedSpecies(),
-  genome_id: 'wheat'
+  genome_id: wheatGenomeInfo.genome_id,
+  url_slug: wheatGenomeInfo.url_slug
 };
 
 const mockState = {
```

Comment:
Can we use `createMockBrowserState` from 'tests/fixtures/browser' instead of building the mock
state here?